### PR TITLE
Make the site breathe a little + responsive

### PR DESCRIPTION
**Pull request by edwardloveall**
Site was like: I.... can't..... breathe......
I was all: It's okay I got you

It responded well.

### CODE REVIEW

File: css/default.css [hunk 1]
```diff
@@ -10,15 +10,15 @@ body {
...
   margin: 0px auto 0px auto;
-  width: 720px;
+  max-width: 720px;
+  padding: 1em;
 }
 
-h1, h2 {
-  padding-bottom: .3em;
-}
+h1, h2 {/* padding-bottom: .3em; */}
```
Comment (gfontenot) [inline css/default.css]: I can just, like, delete this now, right?

Comment (edwardloveall) [inline css/default.css]: Oh, yeah. I'll do that

File: css/default.css [hunk 2]
```diff
@@ -61,7 +59,6 @@ a:hover {
 
 #content {
   padding-top: 1em;
-  border-top: 1px dotted #808000;
```
Comment (gfontenot) [inline css/default.css:64]: is it bad that I like this? Is that bad designering?

Comment (edwardloveall) [inline css/default.css:64]: No, it just felt like an extra separator. The fact that you have your name, large and at the top along with the navigation creates a visual block which separates it from the rest of the page. This dotted line separates it again so I removed it to open it up a little.

Up to you, honestly.

Comment (edwardloveall) [inline css/default.css:64]: What I mean to say is, it's not hurting anything but it could help if you want a certain type of aesthetic.

Comment (gfontenot) [inline css/default.css:64]: I'm going to merge this and then put it back when you aren't looking :yellow_heart:

Comment (gfontenot) [inline css/default.css:64]: yellow heart because they don't have :yolo: and that's close enough

**Comment by gfontenot** [comment]
Merged as 386bcda thaaaaaaaaaaaaanks

**Comment by edwardloveall** [comment]
:+1:

**Pull request closed without merging**